optimum/onnxruntime/modeling_ort.py

```diff
@@ -348,7 +348,7 @@ def infer_onnx_filename(
 
         if len(onnx_files) == 0:
             if fail_if_not_found:
-                raise FileNotFoundError(f"Could not find any ONNX model file in {path}")
+                raise FileNotFoundError(f"Could not find any ONNX model file for the regex {pattern} in {path}.")
```

Comment:
Is it necessary to give the regex?

Reply:
I have no opinion, just thought it's better to pass it for the user to explain why the error is raised.

Reply:
@JingyaHuang What should I do?